aiocqhttp/message.py

```diff
@@ -388,26 +388,28 @@ def iter_function_name_and_extra() -> Iterable[Tuple[str, str]]:
             for cqcode in re.finditer(
                     r'\[CQ:(?P<type>[a-zA-Z0-9-_.]+)'
                     r'(?P<params>'
-                    r'(?:,[a-zA-Z0-9-_.]+=?[^,\]]*)*'
-                    r'),?\]', msg_str):
-                yield 'text', unescape(msg_str[text_begin:cqcode.pos +
-                                               cqcode.start()])
+                    r'(?:,[a-zA-Z0-9-_.]+=[^,\]]+)*'
+                    r'),?\]',
+                    msg_str,
+            ):
+                yield 'text', msg_str[text_begin:cqcode.pos + cqcode.start()]
                 text_begin = cqcode.pos + cqcode.end()
                 yield cqcode.group('type'), cqcode.group('params').lstrip(',')
-            yield 'text', unescape(msg_str[text_begin:])
+            yield 'text', msg_str[text_begin:]
 
         for function_name, extra in iter_function_name_and_extra():
             if function_name == 'text':
                 if extra:
                     # only yield non-empty text segment
                     yield MessageSegment(type_=function_name,
-                                         data={'text': extra})
+                                         data={'text': unescape(extra)})
             else:
                 data = {
-                    k: v for k, v in map(
+                    k: unescape(v) for k, v in map(
```

Comment:
```suggestion
                    k: unescape(''.join(v)) for k, *v in map(
```
是不是可以这样改来支持没有等号的情况...但是看着可读性也太差了